web/src/components/atoms/ContentSection/index.tsx

```diff
@@ -0,0 +1,50 @@
+import styled from "@emotion/styled";
+import { ReactNode } from "react";
+
+export type Props = {
+  title: string;
+  danger?: boolean;
```

Comment:
I don't think the danger is necessary, and makes this atom less reuseable. Please change this to `style` and then pass the border in where necessary.

Reply:
@KaWaite , it only uses two states, the normal and red bordered, so should I made it as `style` and pass, or is it an over burn for this case?

Reply:
Hmmmmm I'll leave this up to you. After you make a decision you can resolve this conversation.

Reply:
@KaWaite , I think for now it is enough, no other border color or style changes will occur, it is better to keep the options limited and clear for now.